src/utils/address/checkAddressPartValid.js

```diff
@@ -4,11 +4,12 @@ import {
   add0x,
 } from '.'
 
-export function checkAddressPartValid(address: string): boolean {
+const CHECK_ADDRESS_VALID_LC = /^0x[0-9a-f]{1,40}$/
+
+export function checkAddressPartValid(address: string) {
   const addr0x: string = add0x(address)
 
-  const isAddressLowerCase: boolean = /^0x[0-9a-f]{1,40}$/.test(addr0x)
-  const isAddressUpperCase: boolean = /^0(x|X)[0-9A-F]{1,40}$/.test(addr0x)
+  const isAddressLowerCase = CHECK_ADDRESS_VALID_LC.test(addr0x.toLowerCase())
 
-  return (isAddressLowerCase || isAddressUpperCase)
+  return isAddressLowerCase
```

Comment:
seems you changed logic of that function.

it checks part of (or full) address without checksum.
- length should be valid
- all symbols should be in the same case lower/upper

Reply:
No, we have only start part of address. For this purpose we have `checkAddressValid` function. This function checks only that string that user typing, can be address.

Reply:
I think it is not possible to get checksum for address part